### DISCUSSION TITLE
Trouble using plane waves traveling through medium with varying velocities in SPECFEM2D

### DISCUSSION THREAD

**Discussion by kdal108** [SPECFEM/specfem3d#1677] [SPECFEM2D General]
I have been working on SPECFEM2D using plane waves through a varying velocity model but I have been finding some issues with doing this. Namely, there appears to be other arrivals generated seemingly from scatterers along the boundary of the dipping low velocity layer (LVL) in my model. I have tried several things to remove these arrivals including having the plane wave start within a homogenous layer prior to reaching the LVL, have a gradual transition from the velocity of the surrounding region to LVL velocity, increase and decrease the grid sizes, shift the plane wave shallower or deeper, changing the incident angle of the plane wave. However, all of these have been unsuccessful. I understand that the plane waves may not be stable in a varying velocity model but is there anything I can do to make this type of modeling work better and remove these arrivals?

Find attached the input files for this modeling in DATA.zip. The figures I've attached are the Z displacement wavefield and the synthetic seismograms. The synthetic waveforms are shown for each station where the left panel is the x component, and right is the z component. Each trace is normalized by the maximum amplitude and station numbers increase from left to right across my study region. The red rectangle and arrow identify the additional arrivals I would like to remove as they overlap with the P waves and PS conversions from the LVL.

Any advice you have would be greatly appreciated.
Thank you!
[DATA.zip](https://github.com/SPECFEM/specfem3d/files/14233695/DATA.zip)
[waveforms_stations.pdf](https://github.com/SPECFEM/specfem3d/files/14233696/waveforms_stations.pdf)
![displacement_wavefieldz](https://github.com/SPECFEM/specfem3d/assets/49471231/f0fa4bea-6058-4d9e-8a9a-a80b5a299971)

**Comment by jpampuero** [comment]
I think it's a good idea to set up the simulation such as the plane wave lies initially inside a single uniform layer.
But what makes you suspect these scattered waves are spurious? The LVL slab in your velocity model can scatter waves. Maybe showing later snapshots or a movie of the wavefield can clarify the origin of these waves ...
